admin_manual/configuration_database/db_conversion.rst

```diff
@@ -69,5 +69,6 @@ Here is a list of known old tables:
 * oc_media_songs
 * oc_media_users
 * oc_permissions
+* oc_privatedata
```

Comment:
It's added by the private data app: https://github.com/nextcloud/privatedata/blob/master/lib/Migration/Version10000Date201707202147.php#L43

Reply:
We removed it from the server core, but added it to an app. Fine by me with stating it here, but maybe add a hint, that it could be added by the privatedata app?

Reply:
I do not fully understand this point, I must let you improve or reject this pull modification. Thanks for all your work!